receiver/hostmetricsreceiver/internal/scraper/processscraper/internal/metadata/generated_metrics.go

```diff
@@ -115,6 +119,59 @@ type metricProcessCPUTime struct {
 	capacity int            // max observed number of data points added to the metric.
 }
 
+type metricProcessCPUPercent struct {
+	data     pmetric.Metric // data buffer for generated metric.
+	settings MetricSettings // metric settings provided by user.
+	capacity int            // max observed number of data points added to the metric.
+}
+
+// init fills process.cpu.percent metric with initial data.
+func (m *metricProcessCPUPercent) init() {
+	m.data.SetName("process.cpu.percent")
+	m.data.SetDescription("Total CPU Percent.")
+	m.data.SetUnit("s")
```

Comment:
I did not get the unit here. Does `s` mean seconds or something else ?

Reply:
Yes, percent should have a different unit, thanks for pointing it out

Reply:
Changed the unit to "%"